### PR TITLE
TEL-4 Implement gathering info about installed Percona packages on host

### PR DESCRIPTION
**Pull request by maxkondr**

### CODE REVIEW

File: metrics/package_test.go [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+		expectedDebian: false,
+		expectedRhel:   true,
+	},
+}
+
+func TestIsDebianFamily(t *testing.T) { //nolint:tparallel
+	t.Parallel()
+
+	for _, tt := range osNames { //nolint:paralleltest
+		t.Run(tt.name, func(t *testing.T) {
```
Comment (artemgavrilov) [inline metrics/package_test.go:104]:
```suggestion
                tt := tt
		t.Run(tt.name, func(t *testing.T) {
		t.Parallel()
```

File: metrics/package_test.go [hunk 2]
```diff
@@ -0,0 +1,250 @@
...
+			require.Equal(t, tt.expectedDebian, isDebianFamily(tt.osName))
+		})
+	}
+}
+
+func TestIsRHELFamily(t *testing.T) { //nolint:tparallel
+	t.Parallel()
+
+	for _, tt := range osNames { //nolint:paralleltest
+		t.Run(tt.name, func(t *testing.T) {
```
Comment (artemgavrilov) [inline metrics/package_test.go:114]:
```suggestion
                tt := tt
		t.Run(tt.name, func(t *testing.T) {
		t.Parallel()
```

File: metrics/package_test.go [hunk 3]
```diff
@@ -0,0 +1,250 @@
...
+			packageName: "percona-xtrabackup-81",
+			dpkgOutput:  "percona-xtrabackup-81 un",
+			dpkgErr:     nil,
+			expectedPkg: nil,
+			expectErr:   errPackageNotFound,
+		},
+	}
+
+	for _, tt := range tests { //nolint:paralleltest
+		t.Run(tt.name, func(t *testing.T) {
```
Comment (artemgavrilov) [inline metrics/package_test.go:177]: Same as above

File: cmd/telemetry-agent/main.go [hunk 4]
```diff
@@ -110,19 +111,35 @@ func processPillarsMetrics(c config.Config) []*metrics.File {
...
+	if len(packages) != 0 {
+		// add info about installed packages to host metrics.
+		jsonData, err := json.Marshal(packages)
+		if err != nil {
+			l.Warnw("failed to marshal installed packages into JSON", zap.Error(err))
+		}
+		hostMetrics.Metrics["installed_packages"] = string(jsonData)
+	}
+
+	l.Debugw("YAKUT", zap.Any("metrics", hostMetrics.Metrics))
```
[the remark this answers is not in the document]
Comment (maxkondr) [reply cmd/telemetry-agent/main.go:141]: oops, done

File: cmd/telemetry-agent/main.go [hunk 5]
```diff
@@ -214,7 +230,7 @@ PERCONA_TELEMETRY_HISTORY_KEEP_INTERVAL - define time interval in seconds for ke
 	}
 	flag.Parse()
 
-	logger.SetupGlobal(&logger.GlobalOpts{LogName: "telemetry-agent", LogDevMode: false, LogDebug: *verbose})
+	logger.SetupGlobal(&logger.GlobalOpts{LogName: "telemetry-agent", LogDevMode: true, LogDebug: *verbose})
```
[the remark this answers is not in the document]
Comment (maxkondr) [reply cmd/telemetry-agent/main.go:233]: done

File: metrics/package.go [hunk 6]
```diff
@@ -0,0 +1,310 @@
...
+		"Percona-Xtradb-Cluster-server-5.7", // deb
+		// PG
+		"percona-postgresql-14", // deb
+		"percona-postgresql-15", // deb
+		"percona-postgresql-16", // deb
+	}
+}
+
+// getRhelPackages returns list of Percona's RHEL specific package names.
+func getRhelPackages() []string {
```
[the remark this answers is not in the document]
Comment (artemgavrilov) [reply metrics/package.go:265]: Probably it makes sense to disable this linter. Not sure what is its purpose.)

**Review by artemgavrilov — APPROVED** [review]
LGTM

**Pull request merged**